compiler/rustc_mir_build/src/build/matches/simplify.rs

```diff
@@ -194,6 +194,12 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
                 Ok(())
             }
 
+            PatKind::Never => {
+                // A never pattern acts like a load from the place.
+                // FIXME(never_patterns): load from the place
```

Comment:
Is it difficult to do this now, or shall we leave this FIXME?

Reply:
I didn't find an easy way, I might need to add a new kind of `Test` or to add a fake kind of binding or something else. I left it for later